spec/index.bs

```diff
@@ -2446,6 +2447,23 @@ enum GPUPowerPreference {
 {{GPURequestAdapterOptions}} has the following members:
 
 <dl dfn-type=dict-member dfn-for=GPURequestAdapterOptions>
+    : <dfn>featureLevel</dfn>
+    ::
+        "Feature level" for the adapter request.
+
+        The allowed <dfn dfn for="">feature level string</dfn> values are:
+
+        <dl dfn-type=dfn dfn-for="feature level string">
+            : <dfn noexport>"core"</dfn>
+            :: No effect.
+            : <dfn noexport>"compatibility"</dfn>
+            :: No effect.
+
+                Note:
+                This value is reserved for future use as a way to opt into additional validation restrictions.
+                Applications should not use this value at this time.
```

Comment:
I think this note is necessary, otherwise the spec is saying that applications can use "compatibility" now and expect to get "core" validation.

Reply:
(FTR, given this, I now have no strong feeling between doing this PR and leaving it as it was before. Corentin's point during the F2F is valid: the spec can disallow passing any value here, and browsers not implementing compat can still go slightly ahead of the spec by allowing-and-ignoring `"compatibility"` knowing that it's the planned string value.)